AUTHORS

```diff
@@ -0,0 +1,35 @@
+# List of AUTHORS who contributed over time to the AzerothCore project
+
+## Warning
+The code of AzerothCore is shipped as it is without any form of warranty,
+and - except for third party libraries licensed
+- under the GPL 2 for old files which you can read from the file "LICENSE-GPL2"
+- under the AGPL 3 for new files which you can read from the file "LICENSE-AGPL3"
+
+## Point of current development
+The project is currently hosted at https://www.azerothcore.org/ and developed under https://github.com/azerothcore
+
+## History of development
+Development of this project dates back to 2004, and was developed under various umbrellas over time:
+* WoW Daemon Team, 2004
+* MaNGOS project, 2005-2008, located at http://www.mangosproject.org
+* MaNGOS project, 2008-2011, located at http://getmangos.com
+* SD2 project, 2008-2009, located at http://getmangos.com
```

Comment:
I'd add UDB too, also SD2 wasn't located at getmangos.com but at www.scriptdev2.com iirc

Reply:
> I'd add UDB too
I do not know what it is :/

Reply:
Replace sd url to http://www.scriptdev2.com/